bridgesync/processor.go

```diff
@@ -72,6 +72,7 @@ type Bridge struct {
 	Amount             *big.Int       `meddler:"amount,bigint" json:"amount"`
 	Metadata           []byte         `meddler:"metadata" json:"metadata"`
 	DepositCount       uint32         `meddler:"deposit_count" json:"deposit_count"`
+	GasTokenAddress    common.Address `meddler:"gas_token_address" json:"gas_token_address"`
```

Comment:
When disucssing should we persist the native token indicator in the database, I wasn't thinking to put the `GasTokenAddress` value, but `IsNativeToken bool`. I think that it is not necessary to persist the GasTokenAddress, as this should be queried only once (on aggkit startup) and we can probably preserve it in memory, e.g. `processor` (as it is defined through an [initializer](https://github.com/agglayer/agg-contracts-internal/blob/b5f9e96eb556c51b80a5799849128eeb81eec6c4/contracts/v2/PolygonZkEVMBridgeV2.sol#L183) and should not be changed).

WDYT?